descriptors/ModuleDescriptor-template.json

```diff
@@ -620,7 +620,12 @@
       "handlers": [
         {
           "methods": ["POST"],
-          "pathPattern": "/_/tenant"
+          "pathPattern": "/_/tenant",
+          "modulePermissions": [
+            "pubsub.event-types.post",
+            "pubsub.publishers.post",
+            "pubsub.subscribers.post"
+          ]
         }, {
```

Comment:
Please add PubSub interfaces dependencies and permissions

Reply:
added